Bottom: efdda925a39817c4e6885a61732a40c3fa9d0fdd
Top:    a978e5871d9236dd445d5cb9372c4e8f18f8ff72
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:21:17 +0100

Refresh of draw-c-add-const-qualifiers-to

---

diff --git a/src/draw.c b/src/draw.c
index 410d15e..f8db2ba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -89,7 +89,7 @@ static void SetPVColor (PinTypePtr, int);
 static void DrawEMark (ElementTypePtr, Coord, Coord, bool);
 static void DrawMask (int side, const BoxType *);
 static void DrawPaste (int side, const BoxType *);
-static void DrawRats (BoxType *);
+static void DrawRats (const BoxType *);
 static void DrawSilk (int side, const BoxType *);
 
 /*--------------------------------------------------------------------------------------
@@ -443,7 +443,7 @@ hole_callback (const BoxType * b, void *cl)
 }
 
 static void
-DrawHoles (bool draw_plated, bool draw_unplated, BoxType *drawn_area)
+DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area)
 {
   int plated = -1;
 
@@ -970,7 +970,7 @@ DrawSilk (int side, const BoxType * drawn_area)
 
 
 static void
-DrawMaskBoardArea (int mask_type, BoxType *drawn_area)
+DrawMaskBoardArea (int mask_type, const BoxType *drawn_area)
 {
   /* Skip the mask drawing if the GUI doesn't want this type */
   if ((mask_type == HID_MASK_BEFORE && !gui->poly_before) ||
@@ -1036,7 +1036,7 @@ DrawPaste (int side, const BoxType *drawn_area)
 }
 
 static void
-DrawRats (BoxTypePtr drawn_area)
+DrawRats (const BoxType *drawn_area)
 {
   /*
    * XXX lesstif allows positive AND negative drawing in HID_MASK_CLEAR.
